asl3-update-nodelist

```diff
@@ -37,11 +37,11 @@ getLastHash() {
 
 writeHash() {
 	curr_hash=$(grep SHA1 $EXTNODES | cut -d "=" -f 2 | tail -n 1)
-	echo "${curr_hash}" > $HASH_FILE
+	#echo "${curr_hash}" > $HASH_FILE
```

Comment:
This is not echoing to stdout, but to a file. Are you sure you want to remove this?

Reply:
Yes, this line should not have been commented out

Reply:
Fixed.